viewer/packages/camera-manager/src/types.ts

```diff
@@ -106,3 +107,14 @@ export const CAMERA_MANAGER_EVENT_TYPE_LIST = ['cameraChange', 'cameraStop'] as
  * Union type of the supported camera manager event types
  */
 export type CameraManagerEventType = (typeof CAMERA_MANAGER_EVENT_TYPE_LIST)[number];
+
+export type NearAndFarPlaneBuffers = {
+  corners: Vector3[];
+  cameraPosition: Vector3;
+  cameraDirection: Vector3;
+};
```

Comment:
I don't think `cameraPosition` and `cameraDirection` need to be part of this type - can't they be derived from the input camera in `CameraManagerHelper`?

Reply:
The reason I'm pedantic on this, is that this type (and the one below) needs to be added to Reveal's API, which I don't want to bloat with many small types

Reply:
Not sure what change you want to see here. Could you clarify? 

Would you prefer to see the parameters spelled out? 

aka

```
static updateCameraNearAndFar(
    camera: PerspectiveCamera,
    boundingBox: Box3,
    cameraPosition: Vector3,
    cameraDirection: Vector3,
    corners: Vector3[]
    ...
  )
```